contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -75,32 +84,75 @@ def _is_javascriptstyle_dir_valid(self, javascriptstyle_dir):
     return True
 
   @memoized_method
-  def _install_javascriptstyle(self, javascriptstyle_dir):
-    with pushd(javascriptstyle_dir):
+  def _bootstrap_default_eslinter(self, bootstrap_dir):
+    with pushd(bootstrap_dir):
       result, yarn_add_command = self.execute_yarnpkg(
-        args=['install'],
+        args=['add', 'eslint'],
```

Comment:
if we are doing a `yarn add eslint` here, maybe we should ping down the eslint version as well?

Reply:
Agreed, added a TODO here as the clean-up should clean up other things as well.